docs/Test.md

```diff
@@ -0,0 +1,76 @@
+Centaur is an integration testing suite for the [Cromwell](http://github.com/broadinstitute/cromwell) execution engine.  It's purpose is to exercise the functionality of a specific deployment of Cromwell, to ensure that it is functioning properly 'in the wild'.  
+
+## Prerequisites
+
+Centaur expects to find a Cromwell server properly configured and running in server mode, listening on port 8000.  This can be configured by modifying the `cromwellUrl` parameter in `application.conf`.
+
+## Running
+
+There are two ways to invoke the intergration tests:
+* `sbt test` - compiles and run via sbt directly, simple but also has the problem of running two times cores tests in parallel which can overwhelm your Cromwell server if running in a development environment
```

Comment:
ToL ok wow this whole file needs some cleanup, not for this PR but at some point